troposphere/ecs.py

```diff
@@ -26,6 +26,20 @@ class DeploymentConfiguration(AWSProperty):
     }
 
 
+class PlacementConstraint(AWSProperty):
+    props = {
+        'Type': (basestring, True),
+        'Expression': (basestring, False),
+    }
+
+
+class PlacementStrategy(AWSProperty):
+    props = {
+        'Type': (basestring, True),
```

Comment:
Yeah, I think there's a mix of validation when it's simple, like in [emr](https://github.com/cloudtools/troposphere/blob/master/troposphere/emr.py#L87-L92), and just a `basestring` in others, so there's not really 100% consistency. IMHO, it's always nice to have early validation, but I can also see the argument that it's more future-proof to let cloudformation be the validator.